packages/nuka/src/hooks/use-paging.tsx

```diff
@@ -12,13 +12,19 @@ type UsePagingReturnType = {
 type PagingProps = {
   totalPages: number;
   wrapMode: CarouselProps['wrapMode'];
+  initialPage?: number;
 };
 
 export function usePaging({
   totalPages,
   wrapMode,
+  initialPage,
 }: PagingProps): UsePagingReturnType {
-  const [currentPage, setCurrentPage] = useState(0);
+  const [currentPage, setCurrentPage] = useState(
```

Comment:
The carousel still visually slides to the given index on load which isn't ideal, but is still better than not having this functionality at all. Not sure if this is blocking though.